### DISCUSSION TITLE
unverified signed commit - signing usage flag

### DISCUSSION THREAD

**Discussion by JaredCorduan** [community/community#21767] [New to GitHub]
<p>I signed a commit <a href="https://github.com/JaredCorduan/ramsey/commit/b6814ed2a099c751d55e3da312bc8e5fab0d14d7" rel="nofollow noopener">here</a>, which is showing up as <strong>Unverified</strong>  with the error:</p>
<pre><code>"The usage flags for the key that signed this don’t allow signing.
GPG key ID: 0151DC52461A815C"
</code></pre>
<p>When I check the key with </p>
<pre><code>$ gpg --list-keys
</code></pre>
<p>it shows that the signing flag is turned on:</p>
<pre><code>pub rsa2048 2013-08-01 [SCEA]
464199634F941439C94F6D8C0151DC52461A815C
</code></pre>
<p>(and there are no subkeys for this key).</p>
<p>Is there anything that I can do to fix this?  Thanks!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Please contact private support at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> so they can take a look at the key attached to your account and dig into this a little deeper.</p>
<p>Thanks for bringing this to our attention!</p>

**Comment by owen9825** [comment]
<p>What ended up happening here? I’m experiencing the same phenomenon.</p>

**Comment by casey** [comment]
<p>I’m having the same issue, all commits in this repository are signed by the same key, but only the first few are showing up as verified:</p>
<p><a href="https://github.com/casey/intermodal/commits/master" rel="nofollow noopener">https://github.com/casey/intermodal/commits/master</a></p>
